src/components/base-address/factories/validation-factory.ts

```diff
@@ -1,6 +1,6 @@
 import { ref } from 'vue-demi'
 
-import { ValidationRule } from '@/enums/address-validation-rules'
+import { ValidationRule } from '@bcrs-shared-components/enums/address-validation-rules'
```

Comment:
If you've add address-validation-rules to enums/index.ts then you can just import the "folder". See how other enums/interfaces are exported/imported.

Reply:
Beautiful, thanks!

What about the 2 interfaces?

Reply:
@PaulGarewal Should the 2 interfaces be in the shared folder?